app/features/import/import-files.ts

```diff
@@ -144,6 +138,15 @@ const executeLoader = (
         onDetailUpdate,
         abortCtl.signal
       )
+      // Wait for the pool to have some data before we signal that
+      // the ingest is done.
```

Comment:
Can we not rely on there being data after `await l.load()` has completed? If not, then we should modify it so it does so consumers of it don't need to worry about this sort of retrying imo

Reply:
We unfortunately can't! I can open an issue on the backend. I spent a half a day debugging this. The load can finish and return, but then if you get the pool stats, the size can be 0 and the span can be null.